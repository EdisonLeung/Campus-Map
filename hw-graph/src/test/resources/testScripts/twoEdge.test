# Creates 2 nodes and Tests creating 2 edges from one node to the other node

# Create graph
CreateGraph twoEdge

# Add 2 nodes
AddNode twoEdge a
AddNode twoEdge b

# adds 2 edges from node a to node b
AddEdge twoEdge a b AB
AddEdge twoEdge a b AB1

# Prints Nodes and children
ListNodes twoEdge
ListChildren twoEdge a
ListChildren twoEdge b