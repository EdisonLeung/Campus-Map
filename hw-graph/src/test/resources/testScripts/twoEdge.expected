# Creates 2 nodes and Tests creating 2 edges from one node to the other node

# Create graph
created graph twoEdge

# Add 2 nodes
added node a to twoEdge
added node b to twoEdge

# adds 2 edges from node a to node b
added edge AB from a to b in twoEdge
added edge AB1 from a to b in twoEdge

# Prints Nodes and children
twoEdge contains: a b
the children of a in twoEdge are: b(AB) b(AB1)
the children of b in twoEdge are: